tempesta_fw/http_parser.c

```diff
@@ -576,9 +576,11 @@ __FSM_STATE(prefix ## _BodyChunkEoL) {					\
 	if (c == '\n') {						\
 		if (parser->to_read)					\
 			__FSM_B_MOVE(prefix ## _BodyReadChunk);		\
-		else							\
+		else {							\
+			msg->body.flags |= TFW_STR_COMPLETE;		\
 			/* Read trailing headers, RFC 2616 3.6.1. */	\
```

Comment:
Please adjust the comment by "RFC 7230 4.1.2" and do so for all outdated 2616 references